### PR TITLE
Feature/deploy site

### LINKED ISSUE #55: Host/Deploy Site
Deploy site to a hosted service

### PR DESCRIPTION
**Pull request by elizasohn**
Adds deploy scripts/ home page name. Some changes have already been pushed automatically to a branch called "gh-pages" that serves the site. Future changes to main should automatically update the deployed site.

You can see the live site here: 
[https://artscii.elizasohn.com/](https://artscii.elizasohn.com/)

### CODE REVIEW

File: artscii/src/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './app/App';
-import reportWebVitals from './reportWebVitals';
+// import reportWebVitals from './reportWebVitals';
```
Comment (unfinishedideas) [inline artscii/src/index.js:5]: Do we need this for anything or could it be deleted?

**Review by unfinishedideas — APPROVED** [review]
Approved! Looks good to me

**Review by srimel — APPROVED** [review]
Nice job!

File: artscii/src/index.js (hunk 1 above)
Comment (elizasohn) [reply artscii/src/index.js:5]: Prob could be deleted unless we care about measuring user activity at some point

**Review by anamika8 — APPROVED** [review]
Looks great to me!

**Pull request merged**